cloudfront/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "oai_iam_arn" {
```

Comment:
Maybe change to output to the following to make it a little clearer: `cloudfront_oai_iam_arn`